woocommerce/google-listings-and-ads
# Minimum WordPress and WooCommerce versions

### Changes proposed in this Pull Request:

This PR bumps the minimum WordPress and WooCommerce versions to 2 versions less then latest, based on [L2 support policy](https://developer.woocommerce.com/what-we-support/)

It also adds a WooCommerce version validator (after plugins are loaded).

Closes #799

### Screenshots:
![image](https://user-images.githubusercontent.com/11388669/122909251-e9e4e700-d34c-11eb-91c1-04fe595d1389.png)

### Detailed test instructions:

1. Activate the plugin on a site with an older version of WordPress (< 5.5)
2. Observe that WordPress gives a warning "Error: Current WordPress version does not meet minimum requirements for Google Listings and Ads."
3. Activate the plugin on a site with an older version of WooCommerce (< 5.2)
2. Observe the admin notice warning the minimum version, none of the GLA pages are available in the menu

### Changelog entry
* Update - Increase required minimum WordPress and WooCommerce versions.
